### DISCUSSION TITLE
Wireless Doorbell camera

### DISCUSSION THREAD

**Discussion by jjvelar** [blakeblackshear/frigate#3728] [Ask A Question]
Hi,
I would like to integrate an EZVIZ doorbell camera in Frigate but as this type of cameras don't generate a stream but only a snapshot when someone triggers it, I'd like to know how go integrate it.

I was thinking in creating a MQTT camera in Home Assistant and Node-red, but as far as I know, MQTT cameras are not supported in Frigate.

Any advice?

**Comment by NickM-27** [comment]
Frigate only works with constant stream devices

**Comment by uvjustin** [comment]
Which model do you have? Most EZVIZ cameras have a RTSP stream with a URL like `rtsp://admin:[YOURPW]@[CAM_IP_ADDR]/?1`
